packages/frontpage/app/(app)/post/[postAuthor]/[postRkey]/[commentAuthor]/[commentRkey]/og-image/route.tsx

```diff
@@ -61,14 +65,25 @@ export async function GET(
         </OgBox>
       </OgBox>
       <OgBottomBar>
-        <img
-          src={avatar}
-          width={48}
-          height={48}
-          style={{
-            borderRadius: "100%",
-          }}
-        />
+        {profile ? (
+          <img
+            src={profile.avatar}
+            width={48}
+            height={48}
+            style={{
+              borderRadius: "100%",
+            }}
+          />
+        ) : (
+          <div
+            style={{
+              width: 48,
+              height: 48,
+              borderRadius: "100%",
+              backgroundColor: "#00000073",
+            }}
+          />
+        )}
```

Comment:
Is it worth extracting this as I see it is also used in `packages/frontpage/app/(app)/post/[postAuthor]/[postRkey]/og-image/route.tsx`